frontend/src/queries/schema.json

```diff
@@ -3752,14 +3752,14 @@
                     "default": {
                         "breakdown_type": "event"
                     },
-                    "description": "Breakdown of the events and actions"
+                    "description": "Breakdown of the events and actions\n\n:TRICKY: We can't set the default to an empty dict, as otherwise datamodel-code-generator does not generate a model with a factory.\n\nThis would cause a different serialization of an empty and a missing prop, which we don't want for stable cache keys.\n\nAs a workaround we simply set any prop to its default, so that a factory is used to instantiate an empty model for a missing prop."
```

Comment:
All these `TRICKY` comments will make it to the public OpenAPI schema as well... maybe there's a way to keep them out.

Reply:
Good point. I moved them to `// commments`, which removes them from the schema.